data/jsons/board_detect.json

```diff
@@ -36,6 +36,28 @@
        "match": [ "SALT BAY" ]
       }
      ]
+    },
+    {
+     "name": "intel-minnow-max-new-gpio-numbering",
```

Comment:
I didn't like the name as well, but nothing better came to my mind. Kind
like yours

On Wed, Oct 7, 2015, 21:14 Gustavo Sverzut Barbieri <
notifications@github.com> wrote:

> In data/jsons/board_detect.json
> <https://github.com/solettaproject/soletta/pull/868#discussion_r41464324>:
>
> > @@ -36,6 +36,28 @@
> >         "match": [ "SALT BAY" ]
> >        }
> >       ]
> > +    },
> > +    {
> > +     "name": "intel-minnow-max-new-gpio-numbering",
>
> awful name, try to add a version or something we can refer. What happens
> if another new schema appears? -new-new? :-D
>
> Add something like intel-minnow-max-linux_gt_3.19.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/solettaproject/soletta/pull/868/files#r41464324>.
>